src/wordpiece_tokenizer.hpp

```diff
@@ -30,14 +27,15 @@ class WordpieceTokenizer : public ov::op::Op {
     );
     WordpieceTokenizer(
         const ov::OutputVector& arguments,
-        const std::shared_ptr<models::FastWordPiece>& tokenizer,
+        const std::shared_ptr<Trie>& trie_root,
+        const std::shared_ptr<Trie>& trie_subwords,
         const std::string& suffix_indicator = "##",
         int max_bytes_per_word = 100
     );
     void validate_and_infer_types() override;
 
     std::shared_ptr<ov::Node> clone_with_new_inputs(const ov::OutputVector& inputs) const override {
-        return std::make_shared<WordpieceTokenizer>(inputs, m_tokenizer, m_suffix_indicator, m_max_bytes_per_word);
+        return std::make_shared<WordpieceTokenizer>(inputs, m_suffix_indicator, m_max_bytes_per_word);
```

Comment:
We should use the other constructor here.

Reply:
done